web/ui/output.svelte

```diff
@@ -38,6 +38,24 @@
       throw e
     }
   }
+
+  export async function triggerCopy(): Promise<void> {
```

Comment:
It is a bad practice creating a special code for test because in storybook you test it and not the real behavior.